src/main/java/seedu/address/model/Model.java

```diff
@@ -87,7 +87,9 @@ public interface Model {
      */
     void setPerson(Person target, Person editedPerson);
 
-    void setPerson(Person target, Person editedPerson, Order order);
+    void setPersonAdd(Person target, Person editedPerson, Order order);
+
+    void setPersonDelete(Person target, Person editedPerson, Order order);
```

Comment:
For this naming maybe you can try:
1. updatePersonAndDeleteOrder
2. setPersonAndDeleteOrder
3. setPersonDeleteOrder
4. refreshPersonDeleteOrder

Reply:
Changed to `setPersonAndDeleteOrder`